utils/node_local_test.py

```diff
@@ -2386,77 +2383,158 @@ def test_cont_clone(self):
-                                           log_check=False,
-                                           label=fn)
-                pt.container_label = fn
-                rc = obj()
-                destroy_container(conf,
-                                  pool.id(),
-                                  pt.container_label,
-                                  valgrind=False,
-                                  log_check=False)
-                pt.container = None
+                ptl.container = create_cont(conf,
+                                            pool.id(),
+                                            ctype="POSIX",
+                                            valgrind=False,
+                                            label=function)
+                ptl.container_label = function
+                rc = test_cb()
+                destroy_container(conf, pool.id(), ptl.container_label, valgrind=False)
+                ptl.container = None
             except Exception as inst:
                 trace = ''.join(traceback.format_tb(inst.__traceback__))
                 duration = time.time() - start
-                conf.wf.add_test_case(pt.test_name,
+                out_wrapper.sprint('{} Failed'.format(function))
+                conf.wf.add_test_case(ptl.test_name,
                                       repr(inst),
+                                      stdout = out_wrapper.get_thread_output(),
                                       output = trace,
                                       test_class='test',
                                       duration = duration)
                 raise
             duration = time.time() - start
-            print('rc from {} is {}'.format(fn, rc))
-            print('Took {:.1f} seconds'.format(duration))
-            conf.wf.add_test_case(pt.test_name,
+            out_wrapper.sprint('Test {} took {:.1f} seconds'.format(function, duration))
+            conf.wf.add_test_case(ptl.test_name,
+                                  stdout = out_wrapper.get_thread_output(),
                                   test_class='test',
                                   duration = duration)
-            if not pt.needs_more:
+            if not ptl.needs_more:
                 break
-            pt.call_index = pt.call_index + 1
+            ptl.call_index = ptl.call_index + 1
 
     server.get_test_pool()
     pool = server.test_pool
 
-    pt = posix_tests(server, conf, pool=pool)
+    out_wrapper = nlt_stdout_wrapper()
+
+    pto = posix_tests(server, conf, pool=pool)
     if test:
         fn = 'test_{}'.format(test)
-        obj = getattr(pt, fn)
+        obj = getattr(pto, fn)
 
-        _run_test()
+        _run_test(ptl=pto, test_cb=obj, function=fn)
     else:
 
-        for fn in sorted(dir(pt)):
+        threads = []
+
+        slow_tests = ['test_readdir_25', 'test_uns_basic', 'test_daos_fs_tool']
+
+        tests = sorted(dir(pto))
+        for test in slow_tests:
```

Comment:
(pylint-redefined-argument-from-local) Redefining argument with the local name 'test'